dokka-subprojects/analysis-kotlin-descriptors-ide/src/main/kotlin/org/jetbrains/kotlin/idea/klib/KlibCompatibilityInfo.kt

```diff
@@ -16,31 +16,24 @@ import java.io.IOException
  */
 internal sealed class KlibCompatibilityInfo(val isCompatible: Boolean) {
```

Comment:
JIC this class can be copy-pasted from https://github.com/JetBrains/intellij-community/blob/master/plugins/kotlin/base/project-structure/src/org/jetbrains/kotlin/idea/base/projectStructure/moduleInfo/KlibCompatibilityInfo.kt

Reply:
Thanks, I will add a reference to this file to the top of the file.
But overall, we don't need `AbstractKlibLibraryInfo`, otherwise, the content is the same